rsfbclient-native/src/varchar.rs

```diff
@@ -35,7 +37,7 @@ impl Varchar {
     pub fn as_bytes(&self) -> &[u8] {
         let len = u16::min(self.capacity, unsafe { self.ptr.as_ref().len }) as usize;
 
-        unsafe { self.ptr.as_ref().data.get_unchecked(..len) }
+        unsafe { self.ptr.as_ref().data.get(..len).unwrap() }
```

Comment:
Running `cargo test -- --test-threads 1`.  To run tests you must have a 'test.fdb' database on your firebird server.

Result:
```
test tests::charset::linking::column_with_charset_none ... thread 'tests::charset::linking::column_with_charset_none' panicked at library/core/src/panicking.rs:219:5:
unsafe precondition(s) violated: slice::from_raw_parts requires the pointer to be aligned and non-null, and the total size of the slice not to exceed `isize::MAX`
```

Backtrace:
```
   0: rust_begin_unwind
             at /rustc/3f5fd8dd41153bc5fdca9427e9e05be2c767ba23/library/std/src/panicking.rs:652:5
   1: core::panicking::panic_nounwind_fmt::runtime
             at /rustc/3f5fd8dd41153bc5fdca9427e9e05be2c767ba23/library/core/src/panicking.rs:110:18
   2: core::panicking::panic_nounwind_fmt
             at /rustc/3f5fd8dd41153bc5fdca9427e9e05be2c767ba23/library/core/src/panicking.rs:120:5
   3: core::panicking::panic_nounwind
             at /rustc/3f5fd8dd41153bc5fdca9427e9e05be2c767ba23/library/core/src/panicking.rs:219:5
   4: core::slice::raw::from_raw_parts::precondition_check
             at /rustc/3f5fd8dd41153bc5fdca9427e9e05be2c767ba23/library/core/src/ub_checks.rs:68:21
   5: core::slice::raw::from_raw_parts
             at /rustc/3f5fd8dd41153bc5fdca9427e9e05be2c767ba23/library/core/src/ub_checks.rs:75:17
   6: <alloc::vec::Vec<T,A> as core::ops::deref::Deref>::deref
             at /rustc/3f5fd8dd41153bc5fdca9427e9e05be2c767ba23/library/alloc/src/vec/mod.rs:2815:18
   7: rsfbclient_native::varchar::Varchar::as_bytes
             at ./rsfbclient-native/src/varchar.rs:40:18
   8: rsfbclient_native::row::ColumnBuffer::to_column
             at ./rsfbclient-native/src/row.rs:173:59
   9: <rsfbclient_native::connection::NativeFbClient<T> as rsfbclient_core::connection::FirebirdClientSqlOps>::fetch::{{closure}}
             at ./rsfbclient-native/src/connection.rs:547:23
```